src/style.css

```diff
@@ -1 +1,37 @@
-/* add styles here */
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  background-image: url(/src/images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars.stars--1 :nth-child(-n + 1) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars.stars--2 :nth-child(-n + 2) {
```

Comment:
Let's combine the selectors with same styles to avoid duplicating yourself

Reply:
all fixed )